middlewares/middlewares.go

```diff
@@ -104,10 +104,34 @@ func AccessControl() negroni.Handler {
 
 // JwtMiddleware check if actual request have JWT
 func JwtMiddleware(key string, algo string) negroni.Handler {
-	jwtMiddleware := jwtmiddleware.New(jwtmiddleware.Options{
-		ValidationKeyGetter: func(token *jwt.Token) (interface{}, error) {
+	var keyGetter jwt.Keyfunc
+	switch algo[0:2]  {
```

Comment:
since this is a public func, this `algo[0:2]` can lead to a possible panic on app runtime, so it'd be best to switch to a more precise approach.

Reply:
@hjelmeland can you finish this feature of supporting another type of token?

Reply:
Is the problem that algo can be shorter that 2 bytes? Could `switch(algo + "  ")[0:2] {` be an acceptable solution?
Sorry, but my bandwidth is pretty much used up these days, and I have already forgotten the little golang I learned this summer. If you prefer #642, that is fine with me. But if you prefer my PF, please say so, and give some guidance , and I can I can rebase and update the PR